tests/eosd_run_test.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+DIR="$(cd "$(dirname "$0")" && pwd)/.."
```

Comment:
While these changes are very good, this whole file is obsolete and slated for removal.  It's being replaced with a suite of python scripts that are much more modular and easy to manage.  You could easily revert this file and submit a new commit, but it's not necessary.  Just be aware that the changes in this file will not survive for long.